packages/dev/gui/src/2D/controls/inputText.ts

```diff
@@ -65,6 +67,42 @@ export class InputText extends Control implements IFocusableControl {
     @serialize()
     public disableMobilePrompt = false;
 
+    /**
+     * Gets or sets outlineWidth of the text to display
+     */
+    public get outlineWidth(): number {
+        return this._outlineWidth;
+    }
+
+    /**
+     * Gets or sets outlineWidth of the text to display
+     */

```

Comment:
I think you don't have to repeat the doc for a get/set pair(?)

Reply:
Was just a copy and paste from the other one. And you are right, it's not really needed